tests/test_cases/test_cocotb/test_handle.py

```diff
@@ -66,21 +67,193 @@ async def test_delayed_assignment_still_errors(dut):
         dut.stream_in_int <= []
 
 
+async def int_values_test(signal, values, limits=_Limits.VECTOR_NBIT):
+    """Test integer access to a signal."""
+    log = logging.getLogger("cocotb.test")
+    for val in values:
+        signal <= val
+        await Timer(10, 'ns')
+
+        if limits == _Limits.VECTOR_NBIT:
+            if val < 0:
+                got = signal.value.signed_integer
+            else:
+                got = signal.value.integer
+        else:
+            got = int(signal)
```

Comment:
`.value` on integer and enum handles should return an `int`.
```suggestion
            got = signal.value
```